apps/platform/src/core/Model.ts

```diff
@@ -277,10 +279,22 @@ export default class Model {
     }
 
     static table(db: Database = App.main.db): Database.QueryBuilder<any> {
-        return db(this.tableName).select(`${this.tableName}.*`)
+        return db(this.tableName)
     }
 
     static raw = raw
+
+    static build<T extends typeof Model>(
```

Comment:
maybe we should always do `${this.tableName}.*` for the static methods other than "query", and use query whenever we want aggregation? just seems a little misleading return types-wise.